src/main/webapp/scripts/map.js

```diff
@@ -200,7 +200,7 @@ function togglePopup(latLng) {
   document.getElementById("popup-add-recs").classList.toggle("active");
   // TODO clear list of events / previously added event listeners
   document.getElementById("submit-recommendation").addEventListener("click", () =>
-    postMarker(latLng));
+    storeMarker(latLng));
 }
```

Comment:
Just to clarify, is this temporary? since, if I am not mistaken,  in the real implementation,  we won't need to storeMarker. the storing and fetching (for marker and recommendation details) will be from and to recommendation entities in datastore

Reply:
Yes, as I explained - this is just to store the marker and the full recommendation. Then, in the next iteration of code, in CL #29 the Marker is removed, and the recommendation is stored (and used to get the lat, lng, id for the markers on the map etc.).